modules/apps/document-library-google-docs/document-library-google-docs/src/main/java/com/liferay/document/library/google/docs/internal/util/GoogleDocsDLFileEntryTypeHelper.java

```diff
@@ -69,6 +70,8 @@ public void addGoogleDocsDLFileEntryType() throws Exception {
 		if (dlFileEntryType == null) {
 			_addGoogleDocsDLFileEntryType(ddmStructure.getStructureId());
 		}
+
```

Comment:
Shouldn't we add the update call in an else clause? If dlFileEntryType == null, the name map will be correctly initialized on add; we would only need to update if it was not null.